scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -489,72 +474,6 @@
 (gauges/defgauge [cook-mesos scheduler front-of-job-queue-mem] (fn [] @front-of-job-queue-mem-atom))
 (gauges/defgauge [cook-mesos scheduler front-of-job-queue-cpus] (fn [] @front-of-job-queue-cpus-atom))
 
-(defn ids-of-backfilled-instances
```

Comment:
I don't think so, the non-backfill functionality already had tests and then there was backfill specific tests that were added. I just removed the backfill ones.